addons/repository.xbmc.org/addon.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="repository.xbmc.org"
 		name="Kodi Add-on repository"
-		version="3.3.1"
+		version="3.4.0"
 		provider-name="Team Kodi">
   <requires>
     <import addon="xbmc.addon" version="12.0.0"/>
   </requires>
 	<extension point="xbmc.addon.repository">
 		<!-- Do not forget to bump add-on version when changing paths to force a repo refresh -->
 		<dir>
-			<info>http://mirrors.kodi.tv/addons/nexus/addons.xml.gz</info>
-			<checksum verify="sha256">http://mirrors.kodi.tv/addons/nexus/addons.xml.gz?sha256</checksum>
-			<datadir>https://mirrors.kodi.tv/addons/nexus</datadir>
-			<artdir>http://mirrors.kodi.tv/addons/nexus</artdir>
+			<info>http://mirrors.kodi.tv/addons/omega/addons.xml.gz</info>
```

Comment:
@yol or @wsnipex can we change these to https whilst we are touching them?

Reply:
so this was intentional since https puts significantly more strain on the mirrors for little benefit here. I'm fine with changing though assuming the infra can deal with it now - @wsnipex?

Reply:
should be fine

Reply:
@glennguy mind changing info, checksum and artdir to https for me.

Reply:
This causes problems because we know not all servers/mirrors actually support https

Reply:
These days, most do, so it's not a big issue. Mirrorbits will only redirect to those that offer https.